plugins/pulp_docker/plugins/registry.py

```diff
@@ -470,7 +470,10 @@ def _get_path(self, path, headers=None):
                 report = self.downloader.download_one(request)
 
         if report.state == report.DOWNLOAD_FAILED:
-            self._raise_path_error(report)
+            if path == '/v2/' and report.error_report.get('response_code') == httplib.UNAUTHORIZED:
```

Comment:
Since this is an unexpected behavior, it's worth adding an in-line comment with a quick explanation and link to the issue.